t_clickhouse/test_clickhouse_logs.py

```diff
@@ -58,7 +58,7 @@ def setUp(self):
             "log_path": tf_cfg.cfg.get("TFW_Logger", "log_path"),
             "clickhouse": {
                 "host": tf_cfg.cfg.get("TFW_Logger", "ip"),
-                "port": tf_cfg.cfg.get("TFW_Logger", "clickhouse_port"),
+                "port": tf_cfg.cfg.get("TFW_Logger", "clickhouse_tcp_port"),
```

Comment:
I forgot remove it... Please remove 57-71 lines. Now the TFW logger config generate automatically.